tox.ini

```diff
@@ -46,7 +47,6 @@ deps =
     # Useful flake8 plugins that are Python and Plone specific:
     flake8-coding
     flake8-debugger
-    flake8-string-format
```

Comment:
Is this intentionally left out?

Reply:
Yes, it creates too much hassle to change all the strings so they match the wishes of this checker.